src/style.css

```diff
@@ -0,0 +1,107 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+:root {
+  --shadow-hover-focus: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+  --linear-hover-focus: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.form__loop {
+  background-image: url("./images/Search.svg");
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.form {
+  width: 100%;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  outline: none;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  color: #3d4e61;
+}
+
+.form-big {
+  position: relative;
+  display: flex;
+  align-items: center;
+  margin: 20px 0;
+}
+
+.form__loop--size_big {
+  width: 19px;
+  height: 19px;
+}
+
+.form-big__big-loop {
+  position: absolute;
+  left: 26px;
+}
+
+.form--input_big {
+  height: 28px;
```

Comment:
usually, it's not a good idea to hardcoding a height. use proper padding and line-height to adjust height

Reply:
check it everywhere